src/SolutionAcceleration.f90

```diff
@@ -0,0 +1,395 @@
+ENDINTERFACE
+
+!> Definition of interface for setInitial and reset
+ABSTRACT INTERFACE
+  SUBROUTINE set_absintfc(solver,x)
+    IMPORT :: SolutionAccelerationType,VectorType
+    CLASS(SolutionAccelerationType),INTENT(INOUT) :: solver
+    CLASS(VectorType),INTENT(INOUT) :: x
+  ENDSUBROUTINE set_absintfc
+ENDINTERFACE
+
+!> @brief the Relaxed Picard Acceleration Type
+TYPE,EXTENDS(SolutionAccelerationType) :: RelaxedPicardType
+  !> Relaxation coefficient
+  REAL(SRK) :: alpha=1.0_SRK
+!
+!List of Type Bound Procedures
+  CONTAINS
+    !> @copybrief RelaxedPicardType::init_RelaxedPicardType
+    !> @copydetails RelaxedPicardType::init_RelaxedPicardType
+    PROCEDURE,PASS :: init => init_RelaxedPicardType
+    !> @copybrief RelaxedPicardType::clear_RelaxedPicardType
+    !> @copydetails RelaxedPicardType::clear_RelaxedPicardType
+    PROCEDURE,PASS :: clear => clear_RelaxedPicardType
+    PROCEDURE,PASS :: clear_picardBase => clear_RelaxedPicardType
+    !> @copybrief RelaxedPicardType::setInitial_RelaxedPicardType
+    !> @copydetails RelaxedPicardType::setInitial_RelaxedPicardType
+    PROCEDURE,PASS :: setInitial => setInitial_RelaxedPicardType
+    PROCEDURE,PASS :: setInitial_picardBase => setInitial_RelaxedPicardType
+    !> @copybrief RelaxedPicardType::step_RelaxedPicardType
+    !> @copydetails RelaxedPicardType::step_RelaxedPicardType
+    PROCEDURE,PASS :: step => step_RelaxedPicardType
+    !> @copybrief RelaxedPicardType::reset_RelaxedPicardType
+    !> @copydetails RelaxedPicardType::reset_RelaxedPicardType
+    PROCEDURE,PASS :: reset => reset_RelaxedPicardType
+ENDTYPE RelaxedPicardType
+
+!> @brief the Relaxed Picard Acceleration Type
+TYPE,EXTENDS(RelaxedPicardType) :: ModifiedPicardType
+  !> Temporary vector to hold previous solution
+  CLASS(VectorType),ALLOCATABLE :: tmpvec
+!
+!List of Type Bound Procedures
+  CONTAINS
+    !> @copybrief ModifiedPicardType::clear_ModifiedPicardType
+    !> @copydetails ModifiedPicardType::clear_ModifiedPicardType
+    PROCEDURE,PASS :: clear => clear_ModifiedPicardType
+    !> @copybrief ModifiedPicardType::setInitial_ModifiedPicardType
+    !> @copydetails ModifiedPicardType::setInitial_ModifiedPicardType
+    PROCEDURE,PASS :: setInitial => setInitial_ModifiedPicardType
+    !> @copybrief ModifiedPicardType::step_ModifiedPicardType
+    !> @copydetails ModifiedPicardType::step_ModifiedPicardType
+    PROCEDURE,PASS :: step => step_ModifiedPicardType
+ENDTYPE ModifiedPicardType
+
+!> Name of module
+CHARACTER(LEN=*),PARAMETER :: modName='SolutionAccelerationModules'
+!
+!===============================================================================
+CONTAINS
+!
+!-------------------------------------------------------------------------------
+!> @brief Initializes the Solution Acceleration base class
+!> @param solver The Solution Acceleration solver to act on
+!> @param Params The options parameter list
+!> @param ce The computing environment to use for the calculation
```

Comment:
should have these in the order they appear in

Reply:
I just did it the same direction @jpjones6 did it :)

Reply:
leave things better than you found them ;)

Reply:
Fixed both here and in Anderson

Reply:
Definitely wouldn't follow my example :P.